arkouda_proxy_server/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "arkouda_proxy_server"
+version = "0.1.0"
```

Comment:
Should the version here match what is in `arkouda_proxy_client/version.py`?

Reply:
@Ethan-DeBandi99 pierce asked the same thing. The problem is that the version cannot match the arkouda tag such as v2023.06.16. The closest I can get is 2023.06.16. As I discussed with @pierce314159, I will look up another way to annotate which arkouda versions are supported in each arkouda_proxy_server release

Reply:
fixed